examples/example-project/LICENSE

```diff
@@ -0,0 +1,48 @@
+
+
+******************************************************************************
+
+THIRD-PARTY SOFTWARE
+
+
+1. express: Express is a web framework focused on developer experience.
```

Comment:
Express is MIT licensed. You should not need to copy the express license into this project.